project_event/models/project_task.py

```diff
@@ -14,6 +14,7 @@ class Task(models.Model):
     _inherit = ['project.task']
```

Comment:
So you reordered the code? it's nice, it's more readable.

Reply:
@eilst